spec/spec_helper.rb

```diff
@@ -2,6 +2,7 @@
 require 'i18n'
 
 I18n.enforce_available_locales = false
+ZendeskAppsSupport::I18n.set_load_path
```

Comment:
I'm probably missing something but It seems like every public method in `I18n` calls `i18n`?